doc/stdenv/stdenv.chapter.md

```diff
@@ -309,13 +309,51 @@ The attribute can also contain a list, a script followed by arguments to be pass
 passthru.updateScript = [ ../../update.sh pname "--requested-release=unstable" ];
 ```
 
-The script will be run with the `UPDATE_NIX_NAME`, `UPDATE_NIX_PNAME`, `UPDATE_NIX_OLD_VERSION` and `UPDATE_NIX_ATTR_PATH` environment variables set respectively to the name, pname, old version and attribute path of the package it is supposed to update.
+Finally, the attribute can be an attribute set, which, in addition to the command, can optionally provide the canonical attribute path to use for the list of extra features the script supports.
+
+```nix
+passthru.updateScript = {
+  command = [ ../../update.sh pname ];
+  attrPath = pname;
+  supportedFeatures = [ … ];
+};
+```
+
+##### How update scripts are executed? {#var-passthru-updateScript-execution}
+
+For information about how to run update scripts, run `nix-shell maintainers/scripts/update.nix` in the root of Nixpkgs repository. The script offers several modes for finding and executing update scripts.
```

Comment:
Which are these modes? Also please link to `update.nix`, otherwise there are additional steps to find out what it actually does, since it's not documented here.